src/main/java/uk/gov/hmcts/reform/sscs/util/SscsOcrDataUtil.java

```diff
@@ -80,10 +80,14 @@ public static String generateDateForCcd(Map<String, Object> pairs, List<String>
     public static String getDateForCcd(String ocrField, List<String> errors, String errorMessage) {
         DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
 
-        try {
-            return LocalDate.parse(ocrField, formatter).toString();
-        } catch (DateTimeParseException ex) {
+        if (ocrField == null) {
```

Comment:
You can leverage Apache Commons `StringUtils.isEmpty(str)` which checks for empty strings and handles null gracefully.

Reply:
Thanks, done.